ox-reveal.el

```diff
@@ -888,7 +888,7 @@ holding export options."
 (defun org-reveal-parse-token (keyword info key &optional value)
   "Return HTML tags or perform SIDE EFFECT according to key.
 Use the previous section tag as the tag of the split section. "
-  (case (intern key)
+  (cl-case (intern key)
```

Comment:
case is already available in Emacs LISP. Still necessary to replace it with cl-case?

Reply:
case is defined by cl.el, and so is deprecated along with it. Try it in `emacs -Q`:

```elisp
(case 'abc
  ((abc def) t))
```

will return an error.

Reply:
I see. Thanks for the clarification